docs/platforms/javascript/common/configuration/integrations/openfeature.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: OpenFeature
+description: "Attaches recent OpenFeature feature flag evaluations to error event context."
+notSupported:
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.bun
+  - javascript.capacitor
+  - javascript.cloudflare
+  - javascript.connect
+  - javascript.cordova
+  - javascript.deno
+  - javascript.electron
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.nodejs
+  - javascript.wasm
+---
+
+<Alert level="info">
+
+This integration only works inside a browser environment.
+
+</Alert>
+
+_Import name: `Sentry.openFeatureIntegration` and `Sentry.OpenFeatureIntegrationHook`_
```

Comment:
maybe we should also add an alert saying this is in beta to match the other docs.

i've been doing this:

```
<Alert level="info" title="Currently in Beta">

The support for **feature flag change tracking** and **feature flag evaluation tracking** is currently in beta.

</Alert>
```